appshared/src/main/java/io/reark/rxgithubapp/shared/viewmodels/RepositoriesViewModel.java

```diff
@@ -84,17 +84,17 @@ public RepositoriesViewModel(@NonNull final GetGitHubRepositorySearch getGitHubR
 
     @NonNull
     public Observable<GitHubRepository> getSelectRepository() {
-        return selectRepository.asObservable();
+        return selectRepository;
```

Comment:
Use `.hide()`

Reply:
Is this useful only for subjects or in some other context too?